src/components/Chat/Chat.tsx

```diff
@@ -794,6 +794,12 @@ function ChatLine(props) {{{
         );
       }
 
+      if (body.substr(0, 9) === '/senseis ') {
```

Comment:
Could we merge this if statement with the /sensei one we don't have duplicated cases?

Perhaps something like `if (/^\/senseis?\s/.test(body))` ?